### ISSUE TITLE
`dotnet --fx-version 1.2.3 path.dll` error message gives invalid path

### ISSUE THREAD

**Issue by AArnott** [dotnet/core-setup#3815]
## Steps to reproduce

Invoke dotnet with an ``-fx-version` argument that refers to a version of the runtime that is not installed:

```
dotnet --fx-version 1.2.3 C:\path\to\netcore\app.dll
```

## Expected  behavior

A sensible error message, that refers me to the location of the installed frameworks: `C:\Program Files\dotnet\shared\Microsoft.NETCore.App`

## Actual behavior

```
It was not possible to find any compatible framework version
The specified framework 'Microsoft.NETCore.App', version '1.2.3' was not found.
  - Check application dependencies and target a framework version installed at:
      \
  - Alternatively, install the framework version '1.2.3'.
```

Notice  the "version installed at \\" error message. That backslash isn't a valid path, but the text leading up to it suggests it should be. This makes the error less actionable.

**Comment by steveharter** [comment]
Moving to future since new features have a high bar at this point for 2.1

**Comment by elinor-fung** [comment]
@sdmaclea I think you already fixed this?

**Comment by sdmaclea** [comment]
Fixed by #5270
